ci/labeler.yaml

```diff
@@ -0,0 +1,238 @@
+# https://github.com/actions/labeler/blob/main/README.md
```

Comment:
Where is this file used? The action in `.github/workflows/` is not pointed to it?

Reply:
The file is used in the ci-lib.
It will behave the same as the current github labeler actions, and therefore I copied the .github/lableler.yaml file as a base.
Forgot to check the comment block.
Will cleanup and clarify in the file

Reply:
I see, understood. I assume there's no easy way to keep the common contents in a single place?

Reply:
Sadly no, the .github/labeler.yaml uses file based 'glob' strings to determine which files have been modified.
The ci/labeler.yaml is using a regular expression, as the matching in the CI-Lib is done based on strings.